flexget/plugins/input/imdb_list.py

```diff
@@ -54,8 +57,9 @@ def on_task_input(self, task, config):
         else:
             url = 'http://www.imdb.com/list/%s' % config['list']
 
-        log.debug('Requesting: %s' % url)
-        page = task.requests.get(url, params=params)
+        headers = {'Accept-Language': config.get('force_language')}
```

Comment:
Does this finally work??? Languages were controlled by ip region for a long time and imdb used to ignore this header.